docstore/awsdynamodb/query_test.go

```diff
@@ -58,7 +58,8 @@ func TestPlanQuery(t *testing.T) {
 		return m
 	}
 
-	cmpopt := cmp.AllowUnexported(dynamodb.ScanInput{}, dynamodb.QueryInput{}, dynamodb.AttributeValue{})
+	// Ignores the ConsistentRead field from both QueryInput and ScanInput.
```

Comment:
Why? What would happen if you did not?

Reply:
This is a dynamo specific test that tests the `planQuery` function only that is only relevant to the expressions.

The `ConsistentRead` field was nil (we didn't set it), now it is a false `*Bool`. The other way is to add that field to every `want` field but we are not testing it and it just makes the test longer. WDYT?

Reply:
Let's try IgnoreFields like we discussed in person.

Reply:
Done.